src/codec.ts

```diff
@@ -1,7 +1,5 @@
 import debug from "debug";
-import { proto_message } from "js-waku";
-import { Decoder, Encoder, Message, ProtoMessage } from "js-waku/lib/interfaces";
-import { MessageV0 } from "js-waku/lib/waku_message/version_0";
+import { DecodedMessage, Decoder, Encoder, proto } from "@waku/core/lib/message/version_0";
```

Comment:
neatpick: `DecodedMessage`, `Decoder` and `Encoder` can be imported from root: `@waku/core`,
`proto` is a re-export from `@waku/proto` (I think).,

Reply:
I decided to use only `implement` and keep initial implementation.